SX1272/SX1272_LoRaRadio.cpp

```diff
@@ -904,6 +896,11 @@ void SX1272_LoRaRadio::receive(uint32_t timeout)
                               RF_RXCONFIG_AGCAUTO_ON |
                               RF_RXCONFIG_RXTRIGER_PREAMBLEDETECT);
 
+            write_to_register(REG_RXTIMEOUT2, _rf_settings.fsk.rx_single_timeout <= 255 ?
```

Comment:
If going to have any special code for this range check, it should produce some sort of diagnostic. The system will fail if you need a timeout bigger than 255 and you can't provide it.

Either the condition won't happen, so it's not worth having the range check, or it might, in which case you want to print something while limping on with a not-big-enough-but-best-we-can-do timeout.

(Probably once only - would be nice if there was a tr_error_once macro. Feature request?)

Reply:
The RX_TIMEOUT is being used for FSK only and with recommended error cushion of 10ms, number of bytes are never going to be more than 255. 100ms value for error cushion is not recommended.

Reply:
So if it's never going to be more than 255, you don't need the check.

Reply:
yeah, just my paranoia .. The types chosen by original developers are weird. I want to have the test, may be I am being a bit too defensive here :) but you taught me that, didn't you ?

Reply:
Paranoia is justified I think here, but this particular "quietly saturate" reaction if they really are out to get you isn't helpful.